packages/gles3/gles3.20160505.alpha/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "gles3"
+version: "20160505.alpha"
+maintainer: "Christophe Raffalli <raffalli@univ-savoie.fr>"
+available:  [ ocaml-version >= "4.02.0" & os != "darwin" ]
+authors:
+  "Christophe Raffalli <raffalli@univ-savoie.fr>, Alexandre Miquel<amiquel@fing.edu.uy>"
```

Comment:
`authors` field is a list so you can use `[ "name" "name2" ]`. With this syntax, both authors will be listed together always and it is difficult to process the metadata.